go/datas/remote_database_handlers.go

```diff
@@ -97,7 +98,7 @@ func createHandler(hndlr Handler, versionCheck bool) Handler {
 		w.Header().Set(NomsVersionHeader, constants.NomsVersion)
 
 		if versionCheck && req.Header.Get(NomsVersionHeader) != constants.NomsVersion {
-			verbose.Log("Returning version mismatch error")
+			log.Printf("returning version mismatch error")
```

Comment:
There's a trade-off to make here. --verbose is all or nothing. 

One approach would be to hide all messages behind verbose and require the embedding program to log all or none.

Another is to identify a small, useful set of messages that will always be printed. I'm proposing the later. Specifically, that request errors should always be printed. 

FWIW, this is exactly why finer grained logging systems are the norm outside of the go universe. When a server embeds software, it's nice for it to have fine control over what to log when.